slash/css/src/Layout/Header/AnchorNavBar/AnchorNavBar.css

```
@@ -0,0 +1,43 @@
+.af-anchor-navbar {
+  width: 100%;
+  background-color: var(--white);
+
+  ul {
+    display: flex;
+    max-width: calc(100% - 112px);
+    margin: auto;
+    padding: 0;
+    align-items: center;
+
+    li {
+      list-style: none;
+
+      .af-anchor-navbar-link {
+        display: flex;
+        padding: 0.8rem 1rem;
+        border-bottom: 4px solid var(--white);
+        align-items: center;
+        gap: 0.5rem;
+        text-decoration: none;
+        color: var(--black);
+
+        &:hover:not(.af-anchor-navbar-link--active),
+        &:focus:not(.af-anchor-navbar-link--active) {
+          border-bottom: 4px solid var(--gray40);
+        }
+      }
```

Pour la gestion des tokens, part sur ce pattern (utiliser le meme pour la color) : 

```suggestion
      .af-anchor-navbar-link {
      --anchor-navbar-border-color: var(--white);
      
        display: flex;
        padding: 0.8rem 1rem;
        border-bottom: 4px solid var(--anchor-navbar-border-color);
        align-items: center;
        gap: 0.5rem;
        text-decoration: none;
        color: var(--black);

        &:hover:not(.af-anchor-navbar-link--active),
        &:focus:not(.af-anchor-navbar-link--active) {
          --anchor-navbar-border-color: var(--gray40);
        }
      }
```

---

done